specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2021-08-01/workbooks_API.json

```diff
@@ -434,13 +437,13 @@
         },
         "kind": {
           "type": "string",
-          "description": "The kind of workbook. Choices are user and shared.",
+          "description": "The kind of workbook. Only valid value is shared.",
           "enum": [
             "user",
             "shared"
           ],
           "x-ms-enum": {
-            "name": "kind",
+            "name": "workbookKind",
```

Comment:
Could you provide the context about the name change?

Reply:
updated the enum list and that caused a differences between this name and the "kind" in myworkbooks_api.json so i had to change it. since we ended up not updating the enum, we could change this back. does the name change affect anything?

Reply:
revert